RELEASES.md

```diff
@@ -6,48 +6,68 @@ Below are the versioning schemes used for publishing debs:
 
 | Build target | Version Format |
 | -------- | -------- |
-| Devel series upstream release | XX.YY |
-| Devel series bugfix release | XX.YY.Z~ubuntu1|
-| Stable series release | XX.YY~ubuntu1~18.04.1|
-| [Daily Build Recipe](https://code.launchpad.net/~canonical-server/+recipe/ua-client-daily) | XX.YY+<revtime>-g<commitish>~ubuntu1~18.04.1 |
+| Devel series upstream release | `XX.YY` |
+| Devel series bugfix release | `XX.YY.Z~ubuntu1`|
+| Stable series release | `XX.YY~ubuntu1~18.04.1`|
+| [Daily Build Recipe](https://code.launchpad.net/~canonical-server/+recipe/ua-client-daily) | `XX.YY+<revtime>-g<commitish>~ubuntu1~18.04.1` |
 | Ubuntu PRO series release | binary copies of Daily PPA |
 
 ## Supported upgrade use-cases based on version formats
 
 | Upgrade path | Version diffs |
-| LTS to LTS | 20.3~ubuntu1~14.04.1 -> 20.3~ubuntu1~16.04.1 |
-| LTS to Daily PPA | 20.3~ubuntu1~14.04.1 -> 20.3+202004011202~ubuntu1~14.04.1 |
-| Ubuntu PRO to latest <series>-updates | 20.3+202004011202~ubuntu1~14.04.1 -> 20.4~ubuntu1~14.04.1 |
-| Ubuntu PRO to Daily PPA | 20.3+202004011202~ubuntu1~14.04.1 -> 20.4+202004021202~ubuntu1~14.04.1 |
+| LTS to LTS | `20.3~ubuntu1~14.04.1 -> 20.3~ubuntu1~16.04.1` |
+| LTS to Daily PPA | `20.3~ubuntu1~14.04.1 -> 20.3+202004011202~ubuntu1~14.04.1` |
+| Ubuntu PRO to latest <series>-updates | `20.3+202004011202~ubuntu1~14.04.1` -> `20.4~ubuntu1~14.04.1` |
+| Ubuntu PRO to Daily PPA | `20.3+202004011202~ubuntu1~14.04.1` -> `20.4+202004021202~ubuntu1~14.04.1` |
 
 
 ## Devel Release Process
 
-Below is the procedure used to release ubuntu-advantage-client to an Ubuntu series:
+Below is the procedure used to release ubuntu-advantage-client to an Ubuntu series, it assumes we already have an annotated tag via `git tag -a 27.1` on the `release-27` branch which has not yet been uploaded to `pkg/ubuntu/devel`:
```

Comment:
There's another related assumption in an comment in the middle of the list. It would help me to see them together up front in the rendered version. Maybe something like this:
```suggestion
Below is the procedure used to release ubuntu-advantage-client to an Ubuntu series. It makes the following assumptions:
* The current ubuntu devel release is impish
* We are releasing version 27.1
* The previous release was 27.0.2
* The commits for 27.0.2 and 27.1 on https://github.com/canonical/ubuntu-advantage-client are tagged with `27.0.2` and `27.1` respectively.
  * These tags should be on the correct commits on the `release-27` branch
  * `27.1` which is the tag of the release we are making, has not yet been uploaded to `pkg/ubuntu/devel`
```